tools/mem-stats-measure.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+JERRY=$1
+TEST=$2
+
```

Comment:
I put this check in `run-perf.sh`, so it is not necessary to check that here too until this script is only used from there. We can place a double check here, but in that case the `is_mem_stats_build` function will be in three different files (`run-perf.sh, mem-stats.sh, mem-stats-measure.sh`).